public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -66,6 +66,33 @@ define(function(require) {
       queryString = queryString === "" ? "?" + cacheBust : queryString + "&" + cacheBust;
       window.location.href = "/projects/new"  + queryString;
     });
+    $('#delete-project-link').click(function() {
+      // TODO: we can do better than this, but let's at least make it harder to lose data.
+      var projectId = Project.getID();
+
+      if(!window.confirm("OK to Delete this project?")) {
+        return false;
+      }
+
+      var request = $.ajax({
+        headers: {
+          "X-Csrf-Token": $("meta[name='csrf-token']").attr("content")
+        },
+        type: "DELETE",
+        url: "/projects/" + projectId
+      });
+      request.done(function() {
+        if(request.status !== 204) {
+          console.error("[Thimble error] sending delete request for project ", projectId, request.status);
+        } else {
+          var queryString = window.location.search;
+          window.location.href = '/projects' + queryString;
```

Comment:
@flukeout if the user clicks "Delete Project" while in the editor, where should we take them?  To their project list, or to the homepage?